config/kube.py

```diff
@@ -1,256 +1,74 @@
-import fcntl
-import os
-import shutil
-import tempfile
-from urllib.parse import urlparse
-
 import click
-import netaddr
-import requests
-import urllib3
-import utilities_common.cli as clicommon
-import yaml
-from sonic_py_common import device_info
-from swsssdk import ConfigDBConnector
-from utilities_common.db import Db
+import socket
```

Comment:
Why did you give this file executable permissions?

Reply:
Not intentionally. I will remove it.

Reply:
Done